aws/resource_aws_prometheus_workspace.go

```diff
@@ -0,0 +1,111 @@
+package aws
+
+import (
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/prometheusservice"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceAwsPrometheusWorkspace() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsPrometheusWorkspaceCreate,
+		Read:   resourceAwsPrometheusWorkspaceRead,
+		Update: resourceAwsPrometheusWorkspaceUpdate,
+		Delete: resourceAwsPrometheusWorkspaceDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"alias": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"status": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
```

Comment:
`status` attribute should be removed unless needed by practitioners.

```suggestion
```

Reply:
I can drop this by introducing waiters. change coming up.